Puppetfile

```diff
@@ -1,26 +1,18 @@
 forge 'https://forgeapi.puppetlabs.com/'
 
-# Dependencies
-# For Puppet 7 support
```

Comment:
can you keep the "puppet 7" comment please? it explains why the commit is pinned (see https://github.com/theforeman/foreman-installer/commit/51d21833ae04aa01d6f1fca7480deb74a41fc0e4)